src/ui/page/error/i18n/error_fr.json

```diff
@@ -0,0 +1,11 @@
+{
+  "error": {
+    "renderingError": {
+      "apology": "Désolé,",
+      "message": "Une erreur s'est produite lors du traitement de votre demande.",
```

Comment:
if we have several errors, `message` key need to be more specific in order to discriminate.

Reply:
I think that we could keep the message `key` as it is the same both for this error and for `not found error`. If you have a more appropriate key name, I can change it :)